frontend/javascripts/admin/organization/pricing_plan_utils.ts

```diff
@@ -83,3 +84,23 @@ export function isFeatureAllowedByPricingPlan(
 
   return isPricingPlanGreaterEqualThan(organization.pricingPlan, requiredPricingPlan);
 }
+
+export function getFeatureNotAvailabeInPlanMessage(
+  requiredPricingPlan: PricingPlanEnum,
+  organization: APIOrganization | null,
+) {
+  let organizationOwnerName = "";
+
+  // expected naming schema for owner: "(M. Mustermann)" | ""
```

Comment:
what is the reasoning for abbreviating the first name? privacy? conciseness?

Reply:
Conciseness. I wanted to keep the tooltip as short as possible.